portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/Staging.testcase

```diff
@@ -1727,7 +1724,7 @@ definition {
 	@description = "This is a use case for LPS-190015, TC1. Publishing a single webcontent by multi publish option."
 	@priority = 3
 	test PublishOneWebContentWithMultiPublishOption {
-		property custom.properties = "feature.flag.LPS-165481=true";
+		property test.liferay.virtual.instance = "false";
```

Comment:
Hey @GaborKomaromi why is `test.liferay.virtual.instance = "false"` necessary now?

Reply:
Deleted them, left them as placeholder as SF was failing due to a line break